apps/node/dal.js

```diff
@@ -0,0 +1,12 @@
+data = {};
```

Comment:
Maybe add a comment to this file saying like "in production, you should use a real database"?